configure.ac

```
@@ -950,6 +950,13 @@ AS_IF([! $arch64],
   [AS_CASE([$ocaml_cv_cc_vendor],
     [gcc-*], [cclibs="$cclibs -latomic"])])
 
+# Support for C11 atomic types
+
+OCAML_CC_SUPPORTS_ATOMIC([$cclibs])
+AS_IF([$cc_supports_atomic],
+  [],
```

Only as I think the `AC_MSG_FAILURE` wants changing below:
```suggestion
AS_IF([! $cc_supports_atomic],
```